app/uk/gov/hmrc/agentaccesscontrol/microserviceWiring.scala

```diff
@@ -31,7 +32,11 @@ import uk.gov.hmrc.play.config.{AppName, RunMode, ServicesConfig}
 import uk.gov.hmrc.play.http.hooks.HttpHook
 import uk.gov.hmrc.play.http.ws._
 
-object WSHttp extends WSGet with WSPut with WSPost with WSDelete with WSPatch with AppName {
+object WSHttp extends WSGet with WSPut with WSPost with WSDelete with WSPatch with AppName with MonitoredWSHttp {
+  val httpAPIs = Map(".*/sa/agents/\\w+/client/\\w+" -> "DES-GetAgentClientRelationship",
+                     ".*/auth/authority" -> "AUTH-GetAuthority",
+                     ".*/auth/oid/\\w+/enrolments" -> "AUTH-GetEnrolments")
```

Comment:
We don't know the URL for enrolments for sure, would it make sense to wrap it explicitly with `monitor` (and use a not monitored WSHttp)?